index.html

```diff
@@ -6,7 +6,325 @@
   </head>
   <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+    <header class="page-header">
+      <h1 class="visually-hidden">Интернет-магазин Девайс</h1> 
+      <a href="index.html"><img src="#" class="header-logo" alt="логотип Девайс"></a>
```

Comment:
Для удобства читаемости кода лучше на 1 строке один тег делать, в данном случае лучше вот такое форматирование по наглядности и упрощает читаемость кода:
```suggestion
      <a href="index.html">
           <img src="#" class="header-logo" alt="логотип Девайс">
      </a>
```